security/backends/elasticsearch/writer.py

```diff
@@ -0,0 +1,328 @@
+from elasticsearch_dsl import Q
+
+from security.config import settings
+from security.enums import (
+    RequestLogState, CeleryTaskInvocationLogState, CeleryTaskRunLogState, CommandState
+)
+from security.backends.signals import (
+    input_request_started, input_request_finished, input_request_error,
+    output_request_started, output_request_finished, output_request_error,
+    command_started, command_output_updated, command_finished, command_error,
+    celery_task_invocation_started, celery_task_invocation_triggered, celery_task_invocation_ignored,
+    celery_task_invocation_timeout, celery_task_invocation_expired,
+    celery_task_run_started, celery_task_run_succeeded, celery_task_run_failed, celery_task_run_retried,
+    celery_task_run_output_updated
+)
+
+from .app import SecurityElasticsearchBackend
+from .models import (
+    InputRequestLog, OutputRequestLog, CommandLog, CeleryTaskRunLog, CeleryTaskInvocationLog, get_key_from_object,
+    get_response_state
+)
+
+
+def set_writer(receiver):
```

Comment:
Strange way to do it. I am afraid, if new signal is added, you might forget to add it here and it will be silently ignored. But if you think this is OK, let it be.

Reply:
Good point classes with abstract methods should be better